comfy/model_management.py

```diff
@@ -650,8 +656,15 @@ def get_free_memory(dev=None, torch_free_too=False):
         dev = get_torch_device()
 
     if hasattr(dev, 'type') and (dev.type == 'cpu' or dev.type == 'mps'):
-        mem_free_total = psutil.virtual_memory().available
-        mem_free_torch = mem_free_total
+        if os.path.isfile('/sys/fs/cgroup/memory/memory.limit_in_bytes'):
+            with open('/sys/fs/cgroup/memory/memory.limit_in_bytes', 'r') as f:
+                mem_used_total = psutil.virtual_memory().used
+                mem_total = int(f.read())
```

Comment:
Since these few lines are repeated above can you put them into their own function?

Reply:
done, i put them in  get_containerd_memory_limit(

Reply:
> Since these few lines are repeated above can you put them into their own function?

done, i put them in get_containerd_memory_limit